src/ontology/mondo-edit.obo

```diff
@@ -514209,6 +514208,21 @@ is_a: MONDO:0859689 {source="https://orcid.org/0000-0001-8134-3037"} ! hepatobil
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0001-5208-3432
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7300" xsd:anyURI
 
+[Term]
+id: MONDO:0100550
+name: orbital myositis
+def: "A rare form of myositis that affects only the orbital muscles." [NCIT:C117296]
+synonym: "ocular myositis" EXACT [https://orcid.org/0000-0002-3458-4839]
```

Comment:
@nicolevasilevsky based on researching to confirm the logical definition, I'd like to propose adding this synonym.

Reply:
sounds good to me, thanks!